wiki.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+mvn -pl fili-wikipedia-example exec:java -Dbard__fili_port=9998 -Dbard__druid_coord=http://localhost:8081/druid/coordinator/v1 -Dbard__druid_broker=http://localhost:8082/druid/v2
```

Comment:
put into am luthier/scripts directory
add cd to ..
Add a comment to describe what it does

Reply:
fixed